public/src/pages/marketplace/Marketplace.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Crystal } from '../../interfaces/Crystal';
+import { getCrystals, getLatestCrystals } from '../../api/httpClient';
+import Header from '../main/Header/Header';
+import Testimonials from './Testimonials/Testimonials';
+
+interface Props {
+  preview: boolean;
+}
+
+export const Marketplace: FC<Props> = (props: Props) => {
+  const [crystals, setCrystals] = useState<Array<Crystal>>([]);
+
+  useEffect(() => {
+    props.preview
+      ? getLatestCrystals().then((data) => setCrystals(data))
+      : getCrystals().then((data) => setCrystals(data));
+  }, []);
+
+  return (
+    <>
+      {props.preview || <Header onInnerPage={true} />}
+
+      <section id="marketplace" className="portfolio">
+        <div className="container" data-aos="fade-up">
+          <div className="section-title marketplaceTitle">
+            <h2>Marketplace</h2>
+          </div>
+
+          {props.preview || (
+            <div className="row">
+              <div className="col-lg-12 d-flex justify-content-center">
+                <ul id="portfolio-flters">
+                  <li data-filter="*" className="filter-active">
+                    All
+                  </li>
+                  <li data-filter=".filter-Healing">Healing</li>
+                  <li data-filter=".filter-Jewellery">Jewellery</li>
+                  <li data-filter=".filter-Gemstones">Gemstones</li>
```

Comment:
Please follow naming conventions
```suggestion
                  <li data-filter=".filter-healing">Healing</li>
                  <li data-filter=".filter-jewellery">Jewellery</li>
                  <li data-filter=".filter-gemstones">Gemstones</li>
```